packages/iris-grid/src/IrisGrid.jsx

```diff
@@ -1024,6 +1024,18 @@ export class IrisGrid extends Component {
     }
   }
 
+  getAlwaysFetchColumns = memoize((alwaysFetchColumns, model) => {
```

Comment:
This isn't going to memoize correctly if the floating columns change after you've initialized (ie. you've changed them from the UI).
Really the memoized function should just take the alwaysFetchColumns and the frozenColumns and join them